netdeploy/remote/nodecfg/nodeConfigurator.go

```diff
@@ -193,16 +205,20 @@ func (nc *nodeConfigurator) registerDNSRecords() (err error) {
 	// to map our network DNS name to our public name (or IP) provided to nodecfg
 	// Network HostName = eg r1.testnet.algodev.network
 	networkHostName := nc.config.Name + "." + string(nc.genesisData.Network) + ".algodev.network"
-	isIP := net.ParseIP(nc.dnsName) != nil
+	isIP := net.ParseIP(nc.DNSName) != nil
 	var recordType string
 	if isIP {
 		recordType = "A"
 	} else {
 		recordType = "CNAME"
 	}
 
-	fmt.Fprintf(os.Stdout, "...... Adding DNS Record '%s' -> '%s' .\n", networkHostName, nc.dnsName)
-	cloudflareDNS.SetDNSRecord(context.Background(), recordType, networkHostName, nc.dnsName, cloudflare.AutomaticTTL, priority, proxied)
+	fmt.Fprintf(os.Stdout, "...... Adding DNS Record '%s' -> '%s' .\n", networkHostName, nc.DNSName)
+	err = cloudflareDNS.SetDNSRecord(context.Background(), recordType, networkHostName, nc.DNSName, cloudflare.AutomaticTTL, priority, proxied)
+	if err != nil {
+		fmt.Fprintf(os.Stdout, "Error creating %s record for %s: %v\n", recordType, nc.DNSName, err)
```

Comment:
`nc` appears to be a nodeConfigurator, which contains

```
	nc := nodeConfigurator{
		config:  cfg,
		dnsName: dnsName,
		DNSOpts: dnsOpts,
	}
```

but in these lines you reference `nc.DNSName`. I think you mean to be referencing DNSOpts structures to get the name. Then again, I could be confused because I wouldn't expect this to compile if `nc.DNSName` was incorrect.